multicompany_property_base/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+==========================
+Multicompany Property Base
+==========================
+
+This module is part of a set of modules that are intended to make sure that
+the property fields are consistent in a multi-company environment.
```

Comment:
please add some content about what the module actually does with some screenshots: This is not straightforward